tests/sentry/replays/scripts/test_delete_replays.py

```diff
@@ -0,0 +1,264 @@
+from __future__ import annotations
+
+import datetime
+from io import BytesIO
+from uuid import uuid4
+from zlib import compress
+
+from sentry.models.file import File
+from sentry.replays.models import ReplayRecordingSegment
+from sentry.replays.scripts.delete_replays import delete_replays
+from sentry.replays.testutils import (
+    mock_replay,
+    mock_rrweb_div_helloworld,
+    mock_segment_console,
+    mock_segment_fullsnapshot,
+    mock_segment_init,
+    mock_segment_nagivation,
+)
+from sentry.testutils.cases import ReplaysSnubaTestCase
+from sentry.utils.json import dumps_htmlsafe
+
+
+class TestDeleteReplays(ReplaysSnubaTestCase):
+    def store_replay_segments(
+        self,
+        replay_id: str,
+        project_id: str,
```

Comment:
I merged a small PR in getsentry yesterday fixing the type of this parameter + line 36. Could you include these changes? https://github.com/getsentry/getsentry/pull/13327

Reply:
Oof.  Forgot to pull before I copied it over.  Thanks I've updated it :D

Reply:
```suggestion
        project_id: int,
```